azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `azure` in the search field.
+. Click *Azure Service Bus Management* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Azure Service Bus Management Connector:
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Azure Service Bus Management Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Azure Service Bus Management* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Azure Service Bus instance. Azure Service Bus Management Connector supports SAS (Shared Access signature) authentication.
+
+To configure the global element for Azure Service Bus Management Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, configure SAS (shared access signature) authentication by entering values for these fields:
```

Comment:
```suggestion
. On the *General* tab, configure Shared Access Signature (SAS) authentication by entering values for these fields:
```

Reply:
To make it consistent with the sentences in your following paragraph of the screenshot explanation,.